__tests__/local.test.js

```diff
@@ -103,6 +103,19 @@ describe('local', () => {
         expect(result).toEqual('qrux');
       });
     });
+
+    it('should return null in callback and promise if data is not set', () => {
+      const spy = jest.fn();
+      testStore.getItem('bar', spy).then((promiseResult) => {
```

Comment:
You're missing a `return` here which makes the test pass without waiting for the promise to resolve

```suggestion
      return testStore.getItem('bar', spy).then((promiseResult) => {
```